docs/romanisim/catalog.rst

```diff
@@ -31,6 +31,36 @@ Following these required fields is a series of columns giving the fluxes of the
 
 The simulator then renders these images in the scene and produces the simulated L1 or L2 images.
 
-The simulator API includes a few simple tools to generate parametric distributions of stars and galaxies.  The ``make_stars`` and ``make_galaxies`` routines make random catalogs of stars and galaxies.  The number of stars and galaxies can be adjusted.  Likewise, the power law index by which the sources' magnitudes are sampled can be adjusted, as can their limiting magnitudes.  Galaxy Sersic parameters, half-light radii, and position angles are chosen at random, with a rough attempt to make brighter galaxies appropriately larger (i.e., conserving surface brightness).  Stars can be chosen to be distributed with a King profile.  This functionality is however very rudimentary and limited, and is better suited for toy problems than real scientific work.  We expect scientific uses to be driven by custom-created catalogs rather than these simple routines.
+The simulator API includes a few simple tools to generate parametric distributions of stars and galaxies.  
+
+The ``make_stars`` and ``make_galaxies`` routines make random catalogs of stars and galaxies.  The number of stars and galaxies can be adjusted.  Likewise, the power law index by which the sources' magnitudes are sampled can be adjusted, as can their limiting magnitudes.  Galaxy Sersic parameters, half-light radii, and position angles are chosen at random, with a rough attempt to make brighter galaxies appropriately larger (i.e., conserving surface brightness).  Stars can be chosen to be distributed with a King profile.  This functionality is however very rudimentary and limited, and is better suited for toy problems than real scientific work.  
+
+We expect scientific uses to be driven by custom-created catalogs rather than the simple routines above, and provide the ``make_cosmos_galaxies`` and ``make_gaia_stars`` routines outlined below to provide that funcionality. 
```

Comment:
```suggestion
We expect scientific uses to be driven by custom-created catalogs rather than the simple routines above.  The routines ``make_cosmos_galaxies`` and ``make_gaia_stars`` outlined below provide some more scientific realism and may be adequate for some use cases.
```

Reply:
Changed